src/tokeniser.hpp

```diff
@@ -58,7 +58,10 @@ class Tokeniser
 
 	/// Whether the next character is to be interpreted as an escape code.
 	/// This usually gets set to true when a backslash is detected.
-	bool escape_next;
+	bool escape_next = false;
+
+	/// Whether the tokeniser is currently in a word.
+	bool in_word = false;
```

Comment:
i'm not sure about doing this - we don't do it anywhere else, and it's not even consistent within this class
also, i tend to feel that it makes it unclear where the variable is set (header vs source file, etc)

revert to the "normal" way of doing things (for now, at least) ?